### PR TITLE
New Message option added to Context Menu

### PR DESCRIPTION
**Pull request by sammyshj**
The New Message option is now added to the Context Menu. On clicking the New Message option, a new tab opens which has a URL, privly-applications/Message/new.html and lets users post new message.

### CODE REVIEW

File: privly.safariextension/privly.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Privly Safari Extension</title>
+  <script type="text/javascript" charset="utf-8">
+    function perform_command(event) {
```
Comment (smcgregor) [inline privly.safariextension/privly.html]: Don't put JS into HTML and make sure you add function/file comments.

Comment (sammyshj) [inline privly.safariextension/privly.html]: I have added another file now scripts/handle_command_event.js which handles the different command events.

File: privly.safariextension/scripts/handle_command_event.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * https://developer.apple.com/library/safari/documentation/Tools/Conceptual/SafariExtensionGuide/SafariExtensionGuide.pdf
+ */
+
+/**
+ * Performs the specific action based on the command event.
+ *
+ * @param event the command event that is dispatched
+ *
+ */
+function perform_command(event) {
```
Comment (smcgregor) [inline privly.safariextension/scripts/handle_command_event.js]: This should probably be performCommand instead of perform_command.

Comment (sammyshj) [inline privly.safariextension/scripts/handle_command_event.js]: Updated the file to use camelCase convention

**Pull request merged**